src/NebulaServiceProvider.php

```diff
@@ -37,6 +38,10 @@ public function boot(): void
                 __DIR__.'/../public' => public_path('vendor/nebula'),
             ], 'public');
 
+            $this->publishes([
+                __DIR__.'/../resources/views' => resource_path('views/vendor/nebula'),
```

Comment:
Same story here^

Reply:
I used `resource_path` here following the previous two statement where you used `config_path` and `public_path` to publish config and asset file respectively. Let me know if you want to use `__DIR__` also